text-notification/index.js

```diff
@@ -0,0 +1,85 @@
+module.exports = class TextNotificationPlugin extends BasePlugin {
+
+    /** Plugin info */
+    static get id()             { return 'zongo.text-notify' }
+    static get name()           { return 'Notify Hosts' }
+    static get description()    { return 'This plugin allows users to notify space hosts via text of their arival in the space' }
+
+    phoneNumberSet = new Set()
+    userName = ''
+    /** Called when the plugin is loaded */
+    onLoad() {
+        this.user.getDisplayName().then(userID => {
+            this.userName = userID
+        }).catch(err => {
+            console.log(err + 'userName is not defined')
+        })
+        // Register button
+        this.menus.register({
+            text: "Notify",
+            section: 'add',
+            adminOnly: false,
```

Comment:
You should add `order: 11` underneath here, so that it appears after the "Chat" and "Emoji" buttons